pkg/autodiscover/autodiscover.go

```diff
@@ -90,23 +89,16 @@ func buildLabelKeyValue(label configuration.Label) (key, value string) {
 // DoAutoDiscover finds objects under test
 func DoAutoDiscover() DiscoveredTestData {
 	var err error
-	data.Env, err = configuration.LoadEnvironmentVariables()
+	data.Env = *configuration.GetTestParameters()
 	if err != nil {
 		logrus.Fatalln("can't load environment variable")
```

Comment:
The function above does not return error anymore, so err here will always be nil. Remove this log message.

Reply:
I forgot that leftover. Good catch! Done.